### PR TITLE
LPS-181586 Object Definition Editor UI is broken if Publications is enabled

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-181586

### CODE REVIEW

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:stop

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:objects

**Comment by alinedoleron** [comment]
ci:test:object

**Comment by rebsilva** [comment]
I'm requesting @naraandrade review for the double check :blush:

**Review by seltong — CHANGES_REQUESTED** [review]

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:object

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/edit_object_definition.jsp [hunk 1]
```diff
@@ -21,10 +21,17 @@ ObjectDefinition objectDefinition = (ObjectDefinition)request.getAttribute(Objec
...
+		<div class="lfr-object__edit-object-definition publication">
+	</c:when>
+	<c:otherwise>
+		<div class="lfr-object__edit-object-definition">
+		</c:otherwise>
+</c:choose>
 	<c:choose>
-		<c:when test='<%= Objects.equals(ParamUtil.getString(request, "screenNavigationCategoryKey"), "details") %>'>
-			<div style="margin-bottom: 4rem;"></div>
+		<c:when test='<%= Objects.equals(ParamUtil.getString(request, "screenNavigationCategoryKey"), "details") || Validator.isNull(ParamUtil.getString(request, "screenNavigationCategoryKey")) %>'>
```
[the remark this answers is not in the document]
Comment (alinedoleron) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/edit_object_definition.jsp]: Thank you, @seltong! I've made the changes =D

**Review by seltong — APPROVED** [review]

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:object

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by alinedoleron** [comment]
ci:forward:force

**Pull request closed without merging**